src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -461,6 +456,12 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         prepID = taskConf.get("PrepID") or self.prepID
         procTask.setPrepID(prepID)
 
+        # has to be done in the very end such that child tasks are set too
+        procTask.setPerformanceMonitor(maxRSS=memoryReq,
+                                       maxVSize=multicore * 20 * 1024,  # ~20GB per core
```

Comment:
shouldn't this be taking from MaxVSize from spec? instead of hard coding ( 20 * 1024 )

Reply:
MaxVSize is supposed to be infinite or something equivalent since we do not really limit on VSize.

Reply:
You're right. I keep confusing it with ImageSize :( I'll just set it to a very high number (1000 * 1024 MiB) in StdBase and I'm in favor of completely removing it in the next month.